python/pyspark/sql/streaming/readwriter.py

```diff
@@ -638,7 +808,25 @@ def format(self, source: str) -> "DataStreamWriter":
 
         Examples
         --------
-        >>> writer = sdf.writeStream.format('json')
+        >>> df = spark.readStream.format("rate").load()
+        >>> df.writeStream.format("text")
+        <pyspark.sql.streaming.readwriter.DataStreamWriter object ...>
```

Comment:
Is this redundant? Looks not related to the example below.

Reply:
the purpose is to show the type of `DataStreamWriter`?

Reply:
Yeah. It's a separate example. In the documentation, it shows like:
<img width="748" alt="Screen Shot 2022-08-11 at 12 16 45 PM" src="https://user-images.githubusercontent.com/6477701/184059362-4e3ed97d-03b8-4a3e-9735-7a6c19f10107.png">